extensions/ql-vscode/src/vscode-tests/minimal-workspace/databases.test.ts

```diff
@@ -31,37 +29,28 @@ describe("databases", () => {
   };
 
   let databaseManager: DatabaseManager;
-  let updateSpy: sinon.SinonSpy;
-  let getSpy: sinon.SinonStub;
-  let dbChangedHandler: sinon.SinonSpy;
-  let registerSpy: sinon.SinonSpy;
-  let deregisterSpy: sinon.SinonSpy;
-  let supportsDatabaseRegistrationSpy: sinon.SinonStub;
-  let supportsLanguageNameSpy: sinon.SinonStub;
-  let resolveDatabaseSpy: sinon.SinonStub;
-
-  let sandbox: sinon.SinonSandbox;
+
+  const updateSpy = jest.fn(() => Promise.resolve());
+  const registerSpy = jest.fn(() => Promise.resolve());
+  const deregisterSpy = jest.fn(() => Promise.resolve());
+  const supportsLanguageNameSpy = jest.fn(() => Promise.resolve(true));
+  const resolveDatabaseSpy = jest.fn(() => Promise.resolve({} as DbInfo));
+
   let dir: tmp.DirResult;
 
   beforeEach(() => {
     dir = tmp.dirSync();
 
-    sandbox = sinon.createSandbox();
-    updateSpy = sandbox.spy();
-    getSpy = sandbox.stub();
-    getSpy.returns([]);
-    registerSpy = sandbox.stub();
-    deregisterSpy = sandbox.stub();
-    dbChangedHandler = sandbox.spy();
-    supportsDatabaseRegistrationSpy = sandbox.stub();
-    supportsDatabaseRegistrationSpy.resolves(true);
-    supportsLanguageNameSpy = sandbox.stub();
-    resolveDatabaseSpy = sandbox.stub();
+    updateSpy.mockReset();
```

Comment:
Since you're calling `mockReset` here, I think this will also clear the implementation given in the `jest.fn` call.

I think you won't need `mockReset` here anymore since the `jest.config.ts` now has `resetMocks: true`, so I think this should be changed to something like:

```suggestion
    updateSpy.mockResolvedValue(undefined);
```

Same thing for all the others.